app/code/Magebit/Faq/view/frontend/templates/question-list.phtml

```diff
@@ -0,0 +1,36 @@
+<?php
+
+/**
+ * Magebit_Faq
+ *
+ * @category     Magebit
+ * @package      Magebit_Faq
+ * @author       Artjoms Fomins <info@magebit.com>
+ * @copyright    Copyright (c) 2023 Magebit, Ltd.(https://www.magebit.com/)
+ */
+
+use Magebit\Faq\Block\QuestionList;
+
+/** @var QuestionList $block
+ */
+?>
+<h1>Frequently Asked Questions</h1>
+<div id="element">
+    <?php foreach ($block->getQuestions() as $question): ?>
+        <div data-role="collapsible" class="faq">
+            <div data-role="trigger" class="faq-title">
+                <span><?= $escaper->escapeHtml(__($question['title'])) ?></span>
```

Comment:
__('') should not be used on values retrieved from the database, as these values can change. It is mostly used for static text